### ISSUE TITLE
Update the instructions for generating an HTML coverage report

### ISSUE THREAD

**Issue by ezzatron** [kahlan/kahlan#334]
On the [Pro Tips](https://kahlan.github.io/docs/pro-tips.html) page, there are instructions to generate an HTML coverage report using Istanbul:

    npm install -g istanbul
    ./bin/kahlan --istanbul="coverage.json"
    istanbul report

The problem is, that the [istanbul](https://www.npmjs.com/package/istanbul) package is abandoned, and has not been maintained for years. The [nyc](https://www.npmjs.com/package/nyc) package is now the standard CLI for Istanbul.

It would be nice to update the docs to show how to generate the HTML with `nyc`. Unfortunately, I had no luck getting it to work 🤷‍♂️

**Comment by jails** [comment]
Indeed good catch, look like the format has been slightly changed, I'll need to figure that out :+1:

**Comment by jails** [comment]
I fixed the it in 4.7.1, I were able to make it work with nyc using the following command lines:

```
mkdir .nyc_output
phpdbg -qrr ./bin/kahlan --istanbul=.nyc_output/coverage.json
nyc report --reporter=html --extension=".php"
google-chrome coverage/index.html
```
